src/Core/Content/Seo/SalesChannel/StoreApiSeoResolver.php

```diff
@@ -48,7 +48,7 @@ public function __construct(
     public static function getSubscribedEvents(): array
     {
         return [
-            KernelEvents::RESPONSE => ['addSeoInformation', 10000],
+            KernelEvents::RESPONSE => ['addSeoInformation', 11000],
```

Comment:
Why is it important the the priority is changed? Does it need to run before or after another specific listener? Then IMHO we should add a comment linking the other subscriber

Reply:
I can do that. Should it be added inline or as doc-block comment?